src/style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background-color: whitesmoke;
+}
+
+section {
+  background-color: #fff;
+  margin: 24px auto;
+  width: 80%;
+  box-shadow: 2px 2px whitesmoke;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  border-bottom: 1px solid gray;
+}
+
+h1 {
+  padding: 1%;
+  font-weight: 300;
+}
+
+button {
+  background-color: white;
+  border: none;
+  padding: 2%;
+}
+
+.add-list {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  border-bottom: 1px solid grey;
+}
+
+input {
+  border: none;
+  background-color: #fff;
+
+  /* width: 80%; */
```

Comment:
You left a commented code in your code base. i suggest you remove it. I t will make your code clean.

Reply:
🙏